### PR TITLE
Readme and license

### LINKED ISSUE #2: Add README
We need a repo readme.

I suggest looking here for inspiration: https://github.com/Cambridge-ICCS/practical-ml-with-pytorch

### LINKED ISSUE #3: Add repo LICENSE
We need a license on this material

I suggest MIT or GPL3 for the code, and CC-BY-SA-NC for the teaching materials.

### PR DESCRIPTION
**Pull request by j-emberton**
Overhaul of README and LICENSE

closes #2 
closes #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+
+# ICCS Scientific visualisation
+
+![GitHub](https://img.shields.io/github/license/Cambridge-ICCS/summer-school-scientific-vis)
+[![CC BY-NC-SA 4.0][cc-by-nc-sa-shield]][cc-by-nc-sa]
+
+This repository contains documentation, resources, and code for the Scientific Visualisation with matplotlib session designed and delivered by [Jack Atkinson](https://jackatkinson.net/) ([**@jatkinson1000**](https://github.com/jatkinson1000))
+and James Emberton ([**@j-emberton**](https://github.com/j-emberton)) of [ICCS](https://github.com/Cambridge-ICCS).  
+This material was originally prepared for the [ICCS](https://iccs.cam.ac.uk/events/iccs-summer-school-2024) 2024 summer school.  
+All materials, including slides and videos, are available such that individuals can cover the course in their own time.
```
Comment (jatkinson1000) [inline README.md:13]: We have no videos, at least not yet.
```suggestion
All materials, including slides and workbooks, are available such that individuals can cover the course in their own time.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+
+The key learning objective from this workshop is to build skills in scientific visualisation through python and its libraries.
+
+However, more specifically we aim to:
+
+summarise key considerations when visualising scientific data,
+provide an understanding of the Matplotlib package and how to use it effectively,
+introduce the Cartopy package and show how it extends Matplotlib functionality for geospatial plotting,
+introduce common climate data formats and show how these can be parsed to generate plottable data,
+and encourage good research software engineering (RSE) practice while using code to generate scientific visualisations.
```
Comment (jatkinson1000) [inline README.md:34]:
```suggestion
- summarise key considerations when visualising scientific data,
- provide an understanding of the Matplotlib package and how to use it effectively,
- introduce the Cartopy package and show how it extends Matplotlib functionality for geospatial plotting,
- introduce common climate data formats and show how these can be parsed to generate plottable data, and
- encourage good research software engineering (RSE) practice while using code to generate scientific visualisations.
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+us before a training session.
+
+## Building
+
+The slides are written using quarto.
+To build them run:
+```
+quarto render Scientific_Vis.qmd 
+```
+from this directory and inspect the output .html file.
```
Comment (jatkinson1000) [inline README.md:88]:
```suggestion
from the `presentation/` directory and inspect the output .html file (assuming you have [Quarto](https://quarto.org/) installed on your machine ).
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+
+
+## Installation and setup
+
+There are three options for participating in this workshop for which instructions are provided below:
+
+* via a [local install](#local-install)
+* on [Google Colab](#google-colab)
+* on [binder](#binder)
```
Comment (jatkinson1000) [inline README.md:97]: I don't think these are set up yet...?
Happy to leave them in, but we should open an issue to make sure we enable them before the summer school.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+Please note that if you have a GitHub account and want to preserve any work you do
+we suggest you first [fork the repository](https://github.com/Cambridge-ICCS/summer-school-scientific-vis/fork) 
+and then clone your fork.
+This will allow you to push your changes and progress from the workshop back up to your
+fork for future reference.
+
+#### 2. Create a virtual environment
+Before installing any Python packages it is important to first create a Python virtual environment.
+This provides an insulated environment inside which we can install Python packages 
+without polluting the operating systems' Python environment.
```
Comment (jatkinson1000) [inline README.md:125]:
```suggestion
without polluting the operating system's Python environment.
```

**Review by jatkinson1000 — APPROVED** [review]
Thanks @j-emberton
This content all looks sensible so I have approved the PR.
I do, however, have a couple of comments and suggestions for you to consider/address before merging.

**Comment by j-emberton** [comment]
Added all those suggestions. Thankyou

File: README.md (hunk 4 above)
Comment (jatkinson1000) [reply README.md:97]: Hi @j-emberton I think this one must have slipped through the cracks.

**Pull request merged**